### ISSUE TITLE
Crash when loading, uh...a structure.

### ISSUE THREAD

**Issue by V4m4cher0n** [TeamChocoQuest/ChocolateQuestRepoured#120]
So, I'm playing a _pretty_ big, 128 mod modpack that I've been working on for a while, and I keep crashing when running into a certain structure. I'm not certain which structure this is, since the game _always_ crashes before it can even come into view, and the logs don't really tell me anything that points to what it could be.

Anywho, here's the **crash report**:
[https://paste.dimdev.org/womuvijana.mccrash](https://paste.dimdev.org/womuvijana.mccrash)

**Comment by DerToaster98** [comment]
Invalid. Mod version used not provided. Modlist not provided. Forge version not provided. Also it doesnt use the issue template. Try updating to the newest mod version first. If it still keeps crashing, update this issue

**Comment by DerToaster98** [comment]
Also we need your CQR config folder

**Comment by V4m4cher0n** [comment]
If you even bothered to simply open the crash report, you could clearly see my Minecraft version, my mod list, and my version of Forge that I'm using. I'm also running the latest version of CQR, which also should be visible in the crash report.

Anyway, now that you mention the CQR config folder, the crash may have something to do with me changing the designated outpost mob to be goblins from the Primitive Mobs mod, which I put down as "primitivemobs:goblin", whereas previously, it said "DEFAULT". I did change a couple other things in the config, that being changing the designated mobs for the cave structures to "quark:ashen", and "quark:dweller", and changing the default block for the volcano to "quark:basalt". When I was testing it earlier though, the changes I made to the volcano structure generated just fine, which leads me to believe that my config changes aren't really the issue, or that either the outpost and cave changes are the culprits. Regardless, I'll test a little more and verify if any of this is the case.

**Comment by LordEpos** [comment]
I don't think we've ever actually tested configuring dummies to be other mods' mobs. In fact, I wasn't aware that was a thing at all. 

While that may work as a whole, I can't find any clear documentation on the primitive mobs goblin, so my shot in the dark guess is that it's not designed to wear armor, or if it is, its doing it in a way that's incompatible with our setup. I'd be willing to test this out myself in greater detail later, but I suspect that this is the source of the trouble.

**Comment by DerToaster98** [comment]
At first: If you want help, it is not a good idea to insult the person trying to help.
Second: I want people to use the issue preset so we have a clear form of issue reports, if makes stuff easier for us. And i do know about the contents of the crash report. I do read them. If you want help, dont insult people trying to help.

Anyways, then we have the cause for that issue. the "dungeonMob" inst a normal mob entry. you can only use mobs specified here: https://cq-repoured.fandom.com/wiki/Dungeon_Inhabitants

Also you should probably read this page: https://cq-repoured.fandom.com/wiki/Dungeon_Configuration (Doesnt cover all dungeon cause its WIP)

Volcano changes should also work just fine cause it actually uses notation like "quark:basalt" for the blocks and the ramp mobs.

And now i think it would probably be a good idea to change the name of "dungeonMob" to a less confusing name. 
To summarize: the issue is your config entry for "dungeonMob" and that "dungeonMob" is a confusing name.

**Comment by V4m4cher0n** [comment]
1. The first bit of my comment wasn't meant to be insulting, but I apologize if it came off that way. 

2. Got it, I'll use the given issue template next time, apologies for the minor inconvenience. 

3. Your help is appreciated, I'll tweak my settings accordingly.
